iocore/net/P_SSLUtils.h

```diff
@@ -60,22 +63,34 @@ class SSLMultiCertConfigLoader
   bool load(SSLCertLookup *lookup);
 
   virtual SSL_CTX *default_server_ssl_ctx();
-  virtual SSL_CTX *init_server_ssl_ctx(std::vector<X509 *> &certList, const SSLMultiCertConfigParams *sslMultCertSettings);
-
-  static bool load_certs(SSL_CTX *ctx, std::vector<X509 *> &certList, const SSLConfigParams *params,
-                         const SSLMultiCertConfigParams *ssl_multi_cert_params);
+  virtual SSL_CTX *init_server_ssl_ctx(std::vector<std::string> const &cert_names_list, std::vector<std::string> const &key_list,
+                                       std::vector<std::string> const &ca_list, std::vector<std::string> &ocsp_list,
+                                       const SSLMultiCertConfigParams *sslMultCertSettings, std::set<std::string> &names);
+
+  static bool load_certs(SSL_CTX *ctx, std::vector<std::string> const &cert_names_list, std::vector<std::string> const &key_list,
```

Comment:
At some point, it's worth while to gather long argument lists like this in a structure, even if it's just a reference tuple. That simplifies the declaration and makes it clearer that the same set of objects is being passed around to the various methods. E.g. something like
```
using CertLoadData = std::tuple<std::vector<std::string> const&, ///< name list
                                                      std::vector<std::string> const&, ///< key list
                                                      std::vector<std::string> const&, ///< CA list
                                                      std::vector<std::string> const&, ///< OSCP list.
                                                      SSLConfigParams*, ///< Configuration parameters.
                                                      SSLMultiCertConfigParams* ///< multicert data
                                                      >;
```
Then the methods are `bool load_certs(SSL_CTX* ctx, CertLoadData data);`

Reply:
Added a new CertLoadData struct under SSLMultiCertConfigLoader